http-service/src/main/java/net/runelite/http/service/chat/ChatController.java

```diff
@@ -100,4 +100,26 @@ public int getKc(@RequestParam String name)
 		}
 		return kc;
 	}
+
+	@PostMapping("/task")
+	public void submitTask(@RequestParam String name, @RequestParam String task)
```

Comment:
Id like for this to probably have name, task, assigned and/or kills left, and location.